### PR TITLE
Smooth hovers was added

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
https://NazariiAlieksieiev.github.io/layout_catalog_hovers/

### CODE REVIEW

File: src/styles/blocks/catalog.scss [hunk 1]
```diff
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/catalog.scss:20]: don't animate all, specify what do you want to

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/styles/blocks/catalog.scss [hunk 2]
```diff
@@ -17,10 +19,11 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-    transition: 300ms;
+    transition: $hovers-time;
 
     &:hover {
-      transform: scale(1.2);
+      width: 240px;
+      height: 360px;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/catalog.scss:26]: should be `transform: scale(1.2);`

File: src/styles/blocks/catalog.scss [hunk 3]
```diff
@@ -17,10 +19,11 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-    transition: 300ms;
+    transition: $hovers-time;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/catalog.scss:22]:
```suggestion
    transition: transform  $hovers-time,  color $hovers-time;
```

Comment (ivanchuk-yurii) [inline src/styles/blocks/catalog.scss:22]:
```suggestion
    transition: transform  $hovers-time,  color $hovers-time;
```

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Review by OlEzhA0 — APPROVED** [review]
Good job

**Pull request closed without merging**